python-lib/powerbi.py

```diff
@@ -179,17 +186,24 @@ def post_table_row(self, rows, dsid, pbi_table=DEFAULT_PBI_TABLE, pbi_group_id=N
                 dsid,
                 pbi_table
             ),
-            data=new_data,  # json.dumps(rows),
-            fail_on_errors=False
+            data=new_data,
+            fail_on_errors=True
         )
         return response
 
     def parse_json_dates(self, rows):
```

Comment:
change name of method now that it does more than just parse dates

Reply:
Done in [b006cc1](https://github.com/dataiku/dss-plugin-microsoft-power-bi/pull/7/commits/b006cc19d00ba74cd3b261c4a9465b1d140edd2a)